rules/do-you-know-the-best-practices-for-frontmatter-in-markdown/rule.md

```diff
@@ -97,3 +99,124 @@ authors:
 ::: good 
 Figure: Good Example {Using arrays helps in listing multiple values under a single key efficiently.} 
 :::
+
+
+
+
+#### Use Meaningful Keys
+ 
+The keys you choose for your Frontmatter should be meaningful and descriptive. They act as identifiers for the associated values, so it's essential that they clearly convey the data they represent.
+ 
+- **Descriptive Names:** Instead of using `desc`, use `description`. Instead of `auth`, use `author`.
+- **Consistency:** Stick to a consistent naming convention, whether it's camelCase, snake_case, or kebab-case.
+ 
+Avoid non-descriptive keys:
+ 
+::: greybox
+```
+---
+t: My Article
+auth: John Doe
+---
+```
+:::
+ 
+::: bad 
+Figure: Bad Example {Shortened or unclear keys can lead to confusion.} 
```

Comment:
Hi @ozairashfaqueSSW 

Unclear about the use of { ... } on your captions...

Reply:
Hi @tiagov8 , 
I have fixed the format. 
Please review again